src/foam/core/console/GridByView.js

```diff
@@ -11,9 +11,110 @@ foam.CLASS({
 
   documentation: 'Table View for GridBy mLang.',
 
+  cssTokens: [
+    {
+      name: 'backgroundTransition',
+      value: 'background-color 0.15s ease'
```

Comment:
Dont need to tokenize this, same in group by, this is unlikely to change in a theme